packages/ember-debug/lib/warn.js

```diff
@@ -18,16 +18,6 @@ export let missingOptionsDeprecation = 'When calling `Ember.warn` you ' +
   '`options` should include an `id` property.';
 export let missingOptionsIdDeprecation = 'When calling `Ember.warn` you must provide `id` in options.';
 
-/**
```

Comment:
This isn't being picked up because this file doesn't declare the module/class/etc that it is for (neither does `Ember.deprecate` but it likely is inheriting the correct info from a file processed before it).